src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -444,8 +505,12 @@ public byte[] ExportEncryptedPkcs8PrivateKey(ReadOnlySpan<byte> passwordBytes, P
         ///   This parameter is treated as uninitialized.
         /// </param>
         /// <returns>
-        ///   A byte array containing the PKCS#8 PrivateKeyInfo representation of the this key.
+        ///   <see langword="true" /> if <paramref name="destination"/> was large enough to hold the result;
+        ///   otherwise, <see langword="false" />.
         /// </returns>
+        /// <exception cref="ArgumentNullException">
+        ///    <paramref name="pbeParameters"/> is <see langword="null"/>.
```

Comment:
Since this change is fixing a lot of one-space under-indentation, this one strikes me as "over indented" by one space.

```suggestion
        ///   <paramref name="pbeParameters"/> is <see langword="null"/>.
```